packages/url_launcher/url_launcher/example/integration_test/url_launcher_test.dart

```diff
@@ -17,10 +17,11 @@ void main() {
 
     // Generally all devices should have some default browser.
     expect(await canLaunch('http://flutter.dev'), true);
+    expect(await canLaunch('https:///www.google.com/404'), true);
 
     // SMS handling is available by default on most platforms.
     if (kIsWeb || !(Platform.isLinux || Platform.isWindows)) {
-      expect(await canLaunch('sms:5555555555'), true);
+      expect(await canLaunch('sms:+12345678900'), true);
```

Comment:
`canLaunch` was giving phone-number-specific answers? That seems very strange.

Reply:
I added an extra test case. This isn't an answer. This is a test case to launch the Message app with this intent to send an SMS to the given phone number.

Reply:
> I added an extra test case. This isn't an answer.

I know it's a test case; the version I reviewed was changing the phone number of the existing case, not adding a new case. I assumed there was a reason for that, rather than it being a random change, and the only reason I could think of would be if `5555555555` was returning false but `+12345678900` were returning true.

> This is a test case to detect if the app can trigger an intent to send an SMS to the given phone number.

If they don't give different results, then I'm not sure what the purpose of the new test case is. Why do we need to test different number formats given that the plugin shouldn't care what the number format is (since it's the OS that parses that)?

Reply:
I thought this was causing problems on Android 30, but it wasn't. The problem was a change in Android 30 about how `resolveActivity` works. 

I removed the test case.